CrowdStrike/crowdstrike_falcon/ingest/parser.yml

```diff
@@ -0,0 +1,138 @@
+          event.kind: event
+          event.type: stop
+          service.name: '{{stage1.message.event.ServiceName}}'
+          event.category: session
+          source.user.id: '{{stage1.message.event.UserId}}'
+        name: set
+  detectionsummaryevent:
+    actions:
+      - set:
+          host.ip: '{{stage1.message.event.LocalIP}}'
+          agent.id: '{{stage1.message.event.AgentIdString}}'
+          host.mac: '{{stage1.message.event.MACAddress}}'
+          file.name: '{{stage1.message.event.FileName}}'
+          file.path: '{{stage1.message.event.FilePath}}'
+          host.name: '{{stage1.message.event.ComputerName}}'
+          user.name: '{{stage.message.event.UserName}}'
+          event.kind: alert
+          event.type: info
+          process.end: '{{stage.message.event.ProcessEndTime}}'
+          process.pid: '{{stage1.message.event.ProcessId}}'
+          file.hash.md5: '{{stage1.message.event.MD5String}}'
+          process.start: '{{stage.message.event.ProcessStartTime}}'
+          event.category: intrusion_detection
+          process.parent.pid: '{{stage1.message.event.ParentProcessId}}'
+          threat.tactic.name: '{{stage1.message.event.Tactic}}'
+          process.hash.sha256: '{{stage1.message.event.SHA256String}}'
+          process.command_line: '{{stage1.message.event.CommandLine}}'
+          crowdstrike.detect_id: '{{stage1.message.event.DetectId}}'
+          threat.technique.name: '{{stage1.message.event.Technique}}'
+          crowdstrike.customer_id: '{{stage1.message.event.CustomerIDString}}'
+          process.parent.executable: '{{stage1.message.event.ParentImageFileName}}'
+          process.parent.command_line: '{{stage1.message.event.ParentCommandLine}}'
+          crowdstrike.detect_description: '{{stage1.message.event.DetectDescription}}'
+        name: set
+  useractivityauditevent:
+    actions:
+      - set:
+          source.ip: '{{stage1.message.event.UserIp}}'
+          event.kind: event
+          event.type: change
+          service.name: '{{stage1.message.event.ServiceName}}'
+          event.category: configuration
+          source.user.id: '{{stage1.message.event.UserId}}'
+          crowdstrike.operation_name: '{{stage1.message.event.OperationName}}'
+        name: set
+  authactivityauditevent:
+    actions:
+      - set:
+          source.ip: '{{stage1.message.event.UserIp}}'
+          event.kind: event
+          event.type: change
+          service.name: '{{stage1.message.event.ServiceName}}'
+          event.category: configuration
+          source.user.id: '{{stage1.message.event.UserId}}'
+          crowdstrike.operation_name: '{{stage1.message.event.OperationName}}'
+        name: set
+  incidentsummaryevent:
+    actions:
+      - set:
+          event.kind: alert
+          event.type: info
+          event.category: intrusion_detection
+          crowdstrike.state: '{{stage1.message.event.State}}'
+          crowdstrike.host_id: '{{stage1.message.event.HostID}}'
```

Comment:
~~Could `host.id` be more suitable?~~
Ok. that's the CrowdStrike identifier of the host; not the identifier of the host